js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -1642,15 +1642,15 @@ usersExplain=Users are the users in the current realm.
-targetHelp=Destination field for the mapper. LOCAL (default) means that the changes are applied to the username stored in local database upon user import. BROKER_ID and BROKER_USERNAME means that the changes are stored into the ID or username used for federation user lookup, respectively.
+targetHelp=Destination field for the mapper. LOCAL (default) means that the changes are applied to the username stored in the local database upon user import. BROKER_ID and BROKER_USERNAME means that the changes are stored into the ID or username used for federation user lookup, respectively.
 setPasswordConfirm=Set password?
 attributeDisplayNameHelp=Display name for the attribute. Supports keys for localized values as well. For example\: ${profile.attribute.phoneNumber}. Hit the 'Globe Route' icon to add translations.
 assignedType=Assigned type
-modeHelp=LDAP_ONLY means that all group mappings of users are retrieved from LDAP and saved into LDAP. READ_ONLY is Read-only LDAP mode where group mappings are retrieved from both LDAP and DB and merged together. New group joins are not saved to LDAP but to DB. IMPORT is Read-only LDAP mode where group mappings are retrieved from LDAP just at the time when user is imported from LDAP and then they are saved to local keycloak DB.
+modeHelp=LDAP_ONLY means that all group mappings of users are retrieved from LDAP and saved into LDAP. READ_ONLY is Read-only LDAP mode where group mappings are retrieved from both LDAP and database and merged together. New group joins are not saved to LDAP but to the database. IMPORT is Read-only LDAP mode where group mappings are retrieved from LDAP just at the time when user is imported from LDAP and then they are saved to the local Keycloak database.
 identityProvider=Identity provider
-forgotPasswordHelpText=Show a link on login page for user to click when they have forgotten their credentials.
+forgotPasswordHelpText=Show a link on the login page for user to click when they have forgotten their credentials.
 identityProviderLinks=Identity provider links
-mapperTypeMsadLdsUserAccountControlMapperHelp=Mapper specific to MSAD LDS. It's able to integrate the MSAD LDS user account state into Keycloak account state (account enabled, password is expired etc). It's using msDS-UserAccountDisabled and pwdLastSet is 0, the Keycloak user is required to update password, if msDS-UserAccountDisabled is 'TRUE' the Keycloak user is disabled as well etc. Mapper is also able to handle exception code from LDAP user authentication.
+mapperTypeMsadLdsUserAccountControlMapperHelp=Mapper specific to MSAD LDS. It is able to integrate the MSAD LDS user account state into Keycloak account state (account enabled, password is expired, and others). It is using msDS-UserAccountDisabled and pwdLastSet is 0, the Keycloak user is required to update password, if msDS-UserAccountDisabled is 'TRUE' the Keycloak user is disabled as well etc. Mapper is also able to handle exception code from LDAP user authentication.
```

Comment:
```suggestion
mapperTypeMsadLdsUserAccountControlMapperHelp=Mapper specific to MSAD LDS. This mapper can integrate the MSAD LDS user account state into the Keycloak account state, such as account enabled or password is expired. It is using msDS-UserAccountDisabled and pwdLastSet is 0, the Keycloak user is required to update password, if msDS-UserAccountDisabled is 'TRUE' the Keycloak user is disabled as well etc. This mapper can also  handle exception codes from LDAP user authentication.
```

Reply:
I believe this sentence needs an "If" at the beginning and it should be broken into two sentences.

Reply:
Thank you for raising the attention to this incomplete sentence. I've updated it in 58726176ba08375c7cf6c46acef76af88a9d431c in a different way that it matches what is currently in the code. Please have another look at this one key "mapperTypeMsadLdsUserAccountControlMapperHelp"